src/components/Evolution.ts

```diff
@@ -35,16 +84,23 @@ export class Evolution<TGameStartr extends FullScreenPokemon> extends Component<
      * Holds evolution requirement checks, keyed by the method of evolution.
      */
     private readonly requirementHandlers: IRequirementHandlers = {
```

Comment:
Regarding this; generic types slimmed them down a bit, and they shouldn't ever have more than 4 or 5 statements in them. Do they still need separate functions?